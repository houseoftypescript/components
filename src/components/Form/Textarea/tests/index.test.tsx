import { render } from '@testing-library/react';
import { Textarea } from '../index';
import React from 'react';

describe('Textarea', () => {
  it('to match snapshot', () => {
    const { container } = render(<Textarea />);
    expect(container).toMatchSnapshot();
  });
});
