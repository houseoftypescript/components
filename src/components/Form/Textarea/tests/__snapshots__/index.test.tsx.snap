// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Textarea to match snapshot 1`] = `
<div>
  <textarea
    class="sc-bcXHqe ejXjgC"
  />
</div>
`;
